libr/include/r_th.h

```diff
@@ -33,6 +35,10 @@ typedef struct r_th_lock_t {
 	R_TH_LOCK_T lock;
 } RThreadLock;
 
+typedef struct r_th_cond_t {
```

Comment:
is a struct needed to just hold 1 value?

Reply:
maybe not, but the content is system-dependent, so using the struct is the safest way to always keep it universal.

Reply:
A define may work too, like its done for R_TH_LOCK_T or R_TH_T